Code/Framework/AzToolsFramework/Tests/Viewport/ViewBookmarkTests.cpp

```diff
@@ -158,9 +154,27 @@ namespace UnitTest
 
         const AZStd::optional<AzToolsFramework::ViewBookmark> lastKnownLocationBookmark = bookmarkInterface->LoadBookmarkAtIndex(4);
 
-        EXPECT_THAT(lastKnownLocationBookmark->m_position, IsClose(cameraPosition));
+        #if AZ_TRAIT_USE_PLATFORM_SIMD_NEON
+        // Expected:  (X: 75,      Y:  0, Z: 64)
+        // Actual:    (X: 74.9989, Y: -0, Z: 64)
+        // Delta:          0.0011      0,     0
+        constexpr float cameraPositionTolerance=0.0012f;
+        #else
+        constexpr float cameraPositionTolerance=0.0f;
+        #endif // AZ_TRAIT_USE_PLATFORM_SIMD_NEON
+        EXPECT_THAT(lastKnownLocationBookmark->m_position, IsCloseTolerance(cameraPosition, cameraPositionTolerance));
+
+        #if AZ_TRAIT_USE_PLATFORM_SIMD_NEON
+        // Expected:  (X: 75,      Y: 0,  Z: 64)
+        // Actual:    (X: 74.9989, Y: -0, Z: 64)
+        // Delta:          0.0011      0,     0
+        constexpr float cameraRotationTolerance=0.0012f;
+        #else
+        constexpr float cameraRotationTolerance=0.0f;
```

Comment:
nitpick: missing whitespace around vfx

Also Does setting the tolerance to 0 perform an exact comparison?
```suggestion
        constexpr float cameraPositionTolerance = 0.0012f;
        #else
        constexpr float cameraPositionTolerance = 0.0f;
        #endif // AZ_TRAIT_USE_PLATFORM_SIMD_NEON
        EXPECT_THAT(lastKnownLocationBookmark->m_position, IsCloseTolerance(cameraPosition, cameraPositionTolerance));

        #if AZ_TRAIT_USE_PLATFORM_SIMD_NEON
        // Expected:  (X: 75,      Y: 0,  Z: 64)
        // Actual:    (X: 74.9989, Y: -0, Z: 64)
        // Delta:          0.0011      0,     0
        constexpr float cameraRotationTolerance = 0.0012f;
        #else
        constexpr float cameraRotationTolerance = 0.0f;
```